src/presentation/pages/Error.vue

```diff
@@ -44,6 +50,17 @@ const message = computed(() => {
 
   return t('errors.default');
 });
+
+useHead({
+  title: t(`pages.error`),
+});
+
+const openPageInfo = {
+  title: te(`errors.${props.code}`) ? t(`errors.${props.code}`) : t(`errors.default`),
```

Comment:
we don't have this i18n token in dictionary. You should add it or use existing key